lib/backend.py

```diff
@@ -5603,8 +5603,13 @@ def DrbdNeedsActivation(disks):
   """
   faulty_disks = []
 
+  lvs_cache = None
```

Comment:
I would write this in a bit different style, with a single assignment and no control flow:

is_plain_disk = conmpat.any(…)
lvs_cache = bdev.LV.GetLvGlobalInfo() if is_plain_disk else None

since I think makes the code 2.5% more readable :), but either way is fine.